Loaddatasets in the tests

mattiasakesson opened the pull request:
## Status

- [ x] Ready
- [ ] Draft
- [ ] Hold

## Description

Change mnist-keras example data to .npz, refactor the scripts in the example to fit the new data format

## Types of changes

What types of changes does your code introduce to FEDn?

- [ ] Hotfix (fixing a critical bug in production)
- [ x] Bugfix
- [ ] New feature
- [ x] Documentation update

## Checklist

_If you're unsure about any of the items below, don't hesitate to ask. We're here to help! 
This is simply a reminder of what we are going to look for before merging your code._

- [ x] This pull request is against **develop** branch (not applicable for hotfixes)
- [ ] I have included a link to the issue on GitHub or JIRA (if any)
- [ ] I have included migration files (if there are changes to the model classes)
- [x ] I have read the [CONTRIBUTING](https://github.com/scaleoutsystems/fedn/blob/master/CONTRIBUTING.md) doc
- [ ] I have included tests to complement my changes
- [ x] I have updated the related documentation (if necessary) 
- [x ] I have added a reviewer for this pull request
- [ x] I have added myself as an author for this pull request

## Further comments

Anything else you think we should know before merging your code!

ahellander reviewed (changes requested):
You need to also update the mnist-pytorch README. Other than that, things look good.

mattiasakesson commented:
> You need to also update the mnist-pytorch README. Other than that, things look good.

Done! I removed the README subsection with loading the data as we now added the data to the repository, same thing for the keras example. I just mention the source of the data in a link.
